pkg/apis/ops/v1alpha1/types.go

```diff
@@ -185,7 +192,6 @@ type TCPHeader struct {
 
 // Packet includes header info.
 type Packet struct {
-	// TODO: change type IPHeader to *IPHeader and correct all internal references
```

Comment:
Out of curiosity, don't we want to change it to pointer in the future?

Reply:
I think so, if you want I can add back the comment, but there are multiple fields I want to change and hope not to add too many TODOs.

Reply:
No strong opinion, just to confirm if it's the case